### PR TITLE
fix Klog flags disppeared

### PR DESCRIPTION
**Pull request by JiahaoWei-RH**
Signed-off-by: JiaHao Wei <1335649406@qq.com>
Fix the Klog flags disppeared issue mentioned in https://github.com/open-cluster-management-io/OCM/issues/31

### CODE REVIEW

File: examples/helloworld/main.go [hunk 1]
```diff
@@ -30,6 +30,8 @@ func main() {
 	pflag.CommandLine.SetNormalizeFunc(utilflag.WordSepNormalizeFunc)
 	pflag.CommandLine.AddGoFlagSet(goflag.CommandLine)
 
+	logs.NewOptions().AddFlags(pflag.CommandLine)
+
 	logs.InitLogs()
 	defer logs.FlushLogs()
```
Comment (zhiweiyin318) [inline examples/helloworld/main.go]: I think the  `logs.InitLogs()
	defer logs.FlushLogs()` are not necessary if we use `logs.NewOptions().AddFlags(pflag.CommandLine)`

Comment (JiahaoWei-RH) [reply examples/helloworld/main.go]: done

**Pull request merged**